app/src/main/java/com/lee989898/soptlee/RepositoryFragment.kt

```diff
@@ -0,0 +1,55 @@
+package com.lee989898.soptlee
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.DiffUtil
+import com.lee989898.soptlee.databinding.FragmentRepositoryBinding
+
+
+class RepositoryFragment : Fragment() {
+
+    private lateinit var repositoryAdapter: RepositoryAdapter
+    private var _binding: FragmentRepositoryBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentRepositoryBinding.inflate(layoutInflater, container, false)
+        initRepositoryAdapter()
+        return binding.root
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
+
+    private fun initRepositoryAdapter() {
+        repositoryAdapter = RepositoryAdapter()
+        binding.repositoryListRv.adapter = repositoryAdapter
+
+        repositoryAdapter.repositoryList.addAll(
+            listOf(
+                RepositoryData(
+                    "안드로이드 과제 레포지토리", "안드로이드 파트 과제안드로이드 파트 과제안드로이드 파트 과제안드로이드 파트 과제" +
+                            "안드로이드 파트 과제안드로이드 파트 과제안드로이드 파트 과제안드로이드 파트 과제안드로이드 파트 과제안드로이드 파트 과제"
```

Comment:
```suggestion
                    "안드로이드 과제 레포지토리".repeat(5)
```

저도 코드리뷰를 받으면서 지난주에 알게된건데, repeat() 메소드를 활용할수도 있더라구요!